src/services/routing.js

```diff
@@ -18,12 +18,11 @@ const routing = {
   getUIDFromSlug(match) {
     return match.params.slug.split('-').pop()
   },
+  getShareUrl(UID) {
+    return `${window.location.origin}/q/${UID}`
```

Comment:
Does Prisma guarantee that the IDs it generates are URL compatible? Maybe the UID should be URL encoded, just in case.

Reply:
Yes, Prisma's IDs are based on [cuid](https://github.com/ericelliott/cuid) which only uses lowercase alphanumeric characters. So it's URL-safe